sync/sync.go

```diff
@@ -265,7 +266,12 @@ func (t *Task) AddReplica(replica string) error {
 		return err
 	}
 
-	if err = t.syncFiles(fromClient, toClient, output.Disks); err != nil {
+	fromServer, err := t.getSourceSyncAgentServerURL()
```

Comment:
We should have this somewhere already? Otherwise client won't be able to connect to the sync agent server.